blog/locale/en/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,50 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
You don't have to generate this file just yet: for English, original values will be used automatically.